orange-and-crane-innovation/ciergio-admin-tool-next-app
# feat(layout): create sidebar component

**Description**
integrate the sidebar into _app.js, for us not to include this element on each page
create 404 page
add some tweaks on tailwind config

**Summary of Changes**
-create sidebar component
-add layout 
-update _app.js to check the layout for specific page base on pathname
-update style 

**How to Test**
-make sure to update the packages by running yarn
-navigate to http://localhost:3000/
